services/kommander/0.3.2/defaults/cm.yaml

```
@@ -86,6 +86,9 @@ data:
           traefik.ingress.kubernetes.io/router.middlewares: "${workspaceNamespace}-stripprefixes@kubernetescrd,${workspaceNamespace}-forwardauth@kubernetescrd"
         path: /dkp/kommander/dashboard
         graphqlPath: /dkp/kommander/dashboard/graphql
+    capimate:
+      image:
+        tag: v2.3.2-dev
```

Is it aligned with kommander version? If so, I would suggest:
```suggestion
        tag: "${kommanderChartVersion:=v2.5.0-dev}"
```

---

Thank you, yes that makes it easy! For release branches, would the default instead be `v2.3.2-dev` (for example for this release branch currently)? And post-release script can automatically bump it to `v2.3.3-dev` after `v2.3.2` is released?